.github/instructions/changelog.instructions.md

```diff
@@ -0,0 +1,621 @@
+  * Server 2022 Gen1/Gen2 – [`AKSWindows-2022-20348.3932.250714`](vhd-notes/AKSWindows/2022/20348.3932.250714.txt).
+  * Server 23H2 Gen1/Gen2 – [`AKSWindows-23H2-25398.1732.250714`](vhd-notes/AKSWindows/23H2/25398.1732.250714.txt).
+```
+
+---
+
+## Link Requirements
+
+### Documentation links
+
+Features and Behavioral changes should include links to relevant documentation.
+
+| Link Type | Format | Example |
+|:----------|:-------|:--------|
+| Microsoft Learn | `[Text](https://learn.microsoft.com/...)` | `[upgrade your node pools](https://learn.microsoft.com/azure/aks/upgrade-cluster)` |
+| GitHub Issues | `[#XXXX](https://github.com/Azure/AKS/issues/XXXX)` or `[GitHub issue](URL)` | `[#4988](https://github.com/Azure/AKS/issues/4988)` |
+| GitHub Releases | `[vX.Y.Z](https://github.com/org/repo/releases/tag/vX.Y.Z)` | `[v1.7.4](https://github.com/Azure/azure-container-networking/releases/tag/v1.7.4)` |
+| CVE References | `[CVE-YYYY-XXXXX](https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXXX)` | `[CVE-2025-22874](https://nvd.nist.gov/vuln/detail/CVE-2025-22874)` |
+| VHD Notes | `` [`ImageName-VERSION`](vhd-notes/ImageType/VERSION.txt) `` | `` [`AKSUbuntu-2204-202510.03.0`](vhd-notes/aks-ubuntu/AKSUbuntu-2204/202510.03.0.txt) `` |
+
+### Link rules
+
+- **No locale in URLs**: Use `https://learn.microsoft.com/azure/...` not `https://learn.microsoft.com/en-us/azure/...`
+- **Descriptive link text**: Use meaningful text, never "click here" or bare URLs
+- **Component versions**: Always use backticks and link to release notes: `` [`v1.33.5`](URL) ``
+- **VHD image names**: Always use backticks and link to VHD notes: `` [`AKSUbuntu-2204-202510.03.0`](vhd-notes/...) ``
+- **VHD notes**: Use relative paths starting with `vhd-notes/` and **validate files exist**
+
+---
+
+## Microsoft Style Guide Compliance
+
+All content MUST follow the Microsoft Style Guide as defined in `.github/copilot-instructions.md`. Key requirements:
+
+### Terminology
+
+| Correct | Incorrect |
+|:--------|:----------|
+| node pool | nodepool, node-pool |
+| cluster autoscaler | Cluster Autoscaler (when not a resource name) |
+| Azure Kubernetes Service (AKS) | Azure Container Service |
+| on-premises | on-premise |
+| cloud-native (adjective) | cloud native (as adjective) |
+| Deployment (resource) | deployment (when referring to the K8s resource) |
+| kubectl | Kubectl |
+
+### Voice and tone
+
+- Use active voice
+- Be direct and concise
+- Use contractions (*it's*, *you're*, *don't*)
+- Front-load important information
+- Write for scanning first, reading second
+
+### Formatting
+
+- Use sentence-style capitalization for headings
+- Use backticks for code, component names, and versions
+- Use bold sparingly for emphasis
+- Keep paragraphs short (3-4 sentences max)
+
+---
+
+## Validation Checklist
+
+Before committing changes to CHANGELOG.md:
+
+- [ ] Release date uses `YYYY-MM-DD` format

```

Comment:
Kubernetes versions - patches, minor versions, deprecations , LTS should all come exclusively in Kubernetes versions section.